public_html/wp-content/plugins/wc-post-types/wc-post-types.php

```diff
@@ -9,6 +9,8 @@
 require_once 'inc/privacy.php';
 require_once 'inc/deprecated.php';
 
+define( 'EP_SESSIONS', 8192 );
```

Comment:
My understanding of rewrite endpoints and EP masks is quite limited, but since this is the next power of 2 after the last mask defined in Core, how likely are we to have a mask collision here with another plugin?

Reply:
I can bump it up, but right now nothing else uses a custom ep_mask in our codebase or installed plugins (and there's no guarantee the other plugin wouldn't do the same).

Reply:
This just seems fragile. I guess it's a limitation in Core though, that there isn't a good way to check for the existence of a custom EP mask integer before registering a new one. And/or dynamically assign the integers for the masks at runtime.

Maybe just add a code comment here explaining what this number means, and linking to EP mask documentation, so it doesn't just look like a magic arbitrary number.

Reply:
I agree, I think it is a core limitation. I'm open to a different approach if you think it's too fragile, otherwise I'll add the comment in.

Reply:
I don't have any other ideas at the moment, so comment it is!

Reply:
Added the comment (and bumped the EP one more power-of-two just in case).